Common/Data/SubscriptionManager.cs

```diff
@@ -209,7 +209,7 @@ public void AddConsolidator(Symbol symbol, IDataConsolidator consolidator)
                 {SecurityType.Cfd, new List<TickType>() { TickType.Quote } },
                 {SecurityType.Future, new List<TickType>() { TickType.Quote, TickType.Trade, TickType.OpenInterest } },
                 {SecurityType.Commodity, new List<TickType>() { TickType.Trade } },
-                {SecurityType.Crypto, new List<TickType>() { TickType.Quote, TickType.Trade } },
+                {SecurityType.Crypto, new List<TickType>() { TickType.Quote } },
```

Comment:
I believe we need both

Reply:
I think if we have access to the trades we should pipe them in by default. I'm thinking of two particular use cases.
1. Ability to use indicators that require volume
2. Ability to better model liquidity/partial fills

Reply:
Agree ideal to have both but backtesting doesn't have the trade data; so it means a stream of error messages for anyone doing backtesting....